packages/suite-desktop/src-electron/libs/getFreePort.ts

```diff
@@ -0,0 +1,15 @@
+import net from 'net';
+
+// TODO: this could be added to @trezor/utils but considering is only for nodejs
```

Comment:
we can create `node-utils` package or smthing like `electron-utils` package, I think we will find lot of things that we can move there

Reply:
That is a great idea!